deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -2510,6 +2510,23 @@ spec:
               appContext:
                 description: App Framework status
                 properties:
+                  appInfoStatus:
+                    description: This represents whether we need to check the status
+                      of apps on remote storage
+                    properties:
+                      lastAppInfoCheckTime:
+                        description: This is set to the time when we get the list
+                          of apps from remote storage.
+                        format: int64
+                        type: integer
+                      needToCheckAfterPollInterval:
+                        description: This is false for the first time when we enter
+                          the reconcile loop. For all other consequent times, this
+                          will be set to true. This will help in checking for the
+                          apps status either right away(for the first time case) or
+                          after the polling interval.
+                        type: boolean
+                    type: object
```

Comment:
I think we don't need this, as we we are tracking in the above 'lastAppInfoCheckTime'